src/lib/Server/Input/Parser/MoveLocation.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+namespace Ibexa\Rest\Server\Input\Parser;
+
+use Ibexa\Contracts\Core\Repository\LocationService;
+use Ibexa\Contracts\Core\Repository\Values\Content\Location;
+use Ibexa\Contracts\Rest\Exceptions;
+use Ibexa\Contracts\Rest\Input\ParsingDispatcher;
+use Ibexa\Rest\Input\BaseParser;
+
+final class MoveLocation extends BaseParser
+{
+    public function __construct(
+        private readonly LocationService $locationService
+    ) {
+    }
+
+    /**
+     * @phpstan-param array{
+     *     'destination': string,
+     * } $data
+     *
+     * @throws \Ibexa\Contracts\Core\Repository\Exceptions\UnauthorizedException
+     * @throws \Ibexa\Contracts\Core\Repository\Exceptions\NotFoundException
+     * @throws \Ibexa\Contracts\Rest\Exceptions\Parser
+     */
+    public function parse(array $data, ParsingDispatcher $parsingDispatcher): Location
+    {
+        if (!array_key_exists('destination', $data)) {
```

Comment:
at some moment we were discussing using symfony validator in cases like this, do we settle on something concrete? 
I know, this seems like a small thing here, but I would rather this be consistent. 

@ibexa/php-dev

Reply:
AFAIR we've agreed to use Symfony Validator indeed, but there was no established pattern.
For instance I've added this to shipping as: https://github.com/ibexa/shipping/blob/main/src/lib/REST/Input/Parser/BaseShipmentModify.php#L111-L116

Reply:
Done, please see https://github.com/ibexa/rest/pull/95/commits/74ebd1195d7578b25175fefb383862a1d2a2dd36